services/forest/src/service/SCIHF.js

```diff
@@ -0,0 +1,150 @@
+module.exports = (SCIHFPersistence, restAPI) => {
+  const SCIHF = {
+    /**
+     * Get values for the forest structural condition index crossed with human footprint
+     * and protected area categories for a given area
+     *
+     * @param {String | Number} areaType area type
+     * @param {String} areaId area id
+     *
+     * @returns {Object[]} Values of Forest Structural Condition Index crossed with human footprint
+     * and protected area categories
+     */
+    getSCIHF: async (areaType, areaId) => {
+      let data;
+      switch (areaType) {
+        case 'ea':
+          data = await SCIHFPersistence.findSCIHFInEA(areaId);
+          break;
+        case 'states':
+          data = await SCIHFPersistence.findSCIHFInState(areaId);
+          break;
+        case 'basinSubzone':
+          data = await SCIHFPersistence.findSCIHFInBasinSubzone(areaId);
+          break;
+        case 'pa': {
+          const binaryProtected = await restAPI.requestBinaryProtectedByCategory(areaId);
+          data = await SCIHFPersistence.findSCIHFInPA(binaryProtected.binary_protected);
+          break;
+        }
+        default:
+          data = null;
+          break;
+      }
+
+      if (!data) {
+        throw new Error('Data for SCIHF doesn\'t exists to the selected area id and area type');
+      }
+
+      const binaryProtectedValues = [...new Set(data.map(b => (b.binary_protected)))].join('&');
+      const categoriesPA = await restAPI.requestCategoriesByBinaryProtected(binaryProtectedValues);
```

Comment:
I'm getting the error `"categoriesPA.find is not a function"`, maybe is a problem with the main service communication, but the possible errors aren't being controlled, either with try/catch or by other mean, the line 43 shouldn't be executed if this doesn't have the desired data

Reply:
The main service is returning
```
{
  code: 'ResourceNotFound',
  message: '/pa/categories/binary_protected/000000000000000&000000000100000&000000010000000&000000010100000&000000100000000&000000100100000&000000110000000&000001000000000&000001010000000&000010000000000&000010100000000&000100000000000&000100000100000&000100010000000&000100100000000&010000000000000 does not exist'
}
```
I think that the rest module should reject the promise is the service is returning an error

Reply:
Apparently there is a problem with the length of the request, don't know exactly why, but the max number of categories i can request is 5.

Reply:
The path param was changed for query param and use ; character as separator